packages/amazonq/src/app/inline/sessionManager.ts

```diff
@@ -24,6 +24,7 @@ export interface CodeWhispererSession {
 export class SessionManager {
     private activeSession?: CodeWhispererSession
     private _acceptedSuggestionCount: number = 0
+    private _refreshedSessions = new Set<string>()
```

Comment:
is it because `maybeRefreshSessionUx` is seldomly invoked so we don't need to clear this set